docs/time/index.rst

```diff
@@ -54,6 +54,19 @@ To create a |Time| object:
   >>> t[1]
   <Time object: scale='utc' format='isot' value=2010-01-01T00:00:00.000>
 
+
+Note: If the ``times`` array in the above example contains a lot of elements, the ``value`` argument will
+display all the elements of the ``times`` array when the |Time| object ``t`` is called.
+To control the number of elements to be displayed, set the ``threshold`` argument with ``np.printoptions`` as follows:
+
+
+    >>> times = np.arange(1000)
+    >>> t = Time(times, format='cxcsec')
+    >>> with np.printoptions(threshold=10):
+    >>>   print(repr(t))
```

Comment:
```suggestion
    >>> with np.printoptions(threshold=10):
    ...     print(repr(t))
```
I think you need to indicate the continuation line inside the block this way; at least that might be the cause for the doctest failures.